package.json

```diff
@@ -52,7 +52,7 @@
   "peerDependencies": {
     "formik": "^2.2.9",
     "react": ">=16",
-    "web-components": "department-of-veterans-affairs/component-library#wc-v0.9.0"
+    "web-components": "*"
```

Comment:
:raised_eyebrow:

Reply:
Oh, I just did this myself on another branch. Switched to Yarn 2 and installed the dependencies. This was an automagic change.

Well, when we publish the web components, this will change to be a proper version spec. :man_shrugging: